src/components/home/Hero.vue

```diff
@@ -18,10 +18,14 @@
         <transition name="slide-up" type="animation" appear>
           <h1 class="title is-1" id="hero-text">
             A module registry for
-            <span id="deno-text">Deno</span>, <br />
+            <!-- <span id="deno-text">Deno</span>, <br /> -->
+            <a class="gradient-text" href="https://deno.land/">
+              Deno
+            </a>
+
             on the
             <a class="gradient-text" href="https://www.arweave.org/">
-              blockchain
+              Blockchain
```

Comment:
```suggestion
              blockweave
```
I don't think that blockchain is a proper noun, though I may be wrong! Also, Arweave is built on a block*weave*, not a block*chain*. Thanks for reminding me to fix this.